ndex2/client.py

```diff
@@ -1051,6 +1534,18 @@ def grant_networks_to_user(self, userid, networkids, permission="READ"):
         """
         Gives read permission to specified networks for the provided user
 
+	.. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		userid = 'INSERT USERNAME HERE'
+		networkids = 'INSERT NETWORK UUID HERE'
+		my_ndex.grant_networks_to_user(userid, networkids, permission="READ")	
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client